openapi/openapi.yaml

```
@@ -357,7 +357,7 @@ components:
           type: string
           minLength: 1
           description: Base image on which the runtime environment should be based on.
-          example: 'registry.access.redhat.com/ubi8/python-36'
+          example: 'quay.io/thoth-station/s2i-thoth-ubi8-py38'
```

To have fully reproducible inspections, it might worth stating exact tag used for testing:

```suggestion
          example: 'quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.32.3'
```